### DISCUSSION TITLE
It says my repository is deleted

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22452] [New to GitHub]
<p>i am trying to accept an assinment but it says my repository is deleted after i refresh the page this it says “There was an error accepting the assignment. We were not able to add the user to your assignment, please try again” what should i do.</p>

**Comment by Zerotask** [comment, marked as answer]
<p>You can contact GitHub directly via <a href="https://support.github.com/contact/feedback" class="inline-onebox" rel="noopener nofollow ugc">Share feedback - GitHub Support</a></p>
